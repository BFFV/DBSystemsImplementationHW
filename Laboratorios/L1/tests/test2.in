11
w catorce
w dos
f
w tres
x
w uno
r 3
r 1
r 2
x
r 0
f
