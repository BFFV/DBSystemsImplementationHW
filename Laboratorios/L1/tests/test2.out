0
1
2
3
uno
dos
tres
catorce
